tools/dump_tool/dump_tool.h

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************
+*
+* Copyright 2020 Samsung Electronics All Rights Reserved.
+*
+* File Name : dump_tool.h
+* Description: Receive FS and Ram Dump using UART
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+******************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <errno.h>
```

Comment:
can you check only necessary headers included here...

Reply:
Done